site/content/overview/custom-policies.md

```diff
@@ -0,0 +1,159 @@
+These custom policies follow the Gateway API's pattern of [Policy Attachment](https://gateway-api.sigs.k8s.io/reference/policy-attachment/), which allows users to extend the Gateway API functionality by creating implementation-specific policies and attaching them to Kubernetes objects such as HTTPRoutes, Gateways, and Services.
+
+Policies are a Kubernetes object that augments the behavior of an object in a standard way. Policies can be attached to one object ([Direct Policy Attachment](#direct-policy-attachment)) or objects in a hierarchy ([Inherited Policy Attachment](#inherited-policy-attachment)).
+The following table summarizes NGINX Gateway Fabric custom policies:
+
+{{< bootstrap-table "table table-striped table-bordered" >}}
+
+| Policy                                                               | Description                                             | Attachment Type | Supported Target Object(s)    | Supports Multiple Target Refs | Mergeable | API Version |
+|----------------------------------------------------------------------|---------------------------------------------------------|-----------------|-------------------------------|-------------------------------|-----------|-------------|
+| ClientSettingsPolicy                                                 | Configure connection behavior between client and NGINX  | Inherited       | Gateway, HTTPRoute, GRPCRoute | No                            | Yes       | v1alpha1    |
+| [ObservabilityPolicy]({{<relref "/how-to/monitoring/tracing.md" >}}) | Define settings related to tracing, metrics, or logging | Direct          | HTTPRoute, GRPCRoute          | Yes                           | No        | v1alpha1    |
+
+
+{{< important >}}
+NGINX Gateway Fabric policies do not work with [HTTPRoute matches](https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io/v1.HTTPRouteMatch) with `headers`, `params`, or `method` matchers defined. This will be addressed in a future release.
+{{< /important >}}
+
+## Terminology
+
+- _Attachment Type_. How the policy attaches to an object. Attachment type can be "direct" or "inherited".
+- _Supported Target Object(s)_. API objects the policy can be applied to.
+- _Supports Multiple Target Refs_. Whether a single policy can target multiple objects.
+- _Mergeable_. Whether policies that target the same object can be merged.
+
+## Direct Policy Attachment
+
+A Direct Policy Attachment is a policy that references a single object -- such as a Gateway or HTTPRoute. It is tightly bound to one instance of a particular Kind within a single Namespace or an instance of a single Kind at the cluster-scope. It affects _only_ the object specified in its TargetRef.
+
+This diagram uses a fictional retry policy to show how Direct Policy Attachment works:
+
+{{<img src="img/direct-policy-attachment.png" alt="Direct Policy Attachment">}}
```

Comment:
```suggestion
{{<img src="img/direct-policy-attachment.png" alt="">}}
```
Alternative text is meant to be a replacement for an image, not a title stating what it is.

An empty alt tag has specific meaning to a browser (Distinctly different to a "missing" one): the alt text needs to be a complete description of the image or the paragraph underneath must include all of the relevant information.